### DISCUSSION TITLE
如何改变post header中的时间日期格式

### DISCUSSION THREAD

**Discussion by realzza** [ppoffice/hexo-theme-icarus#1057] [Q&A] [answered]
在调整blog的head格式的时候，发布时间和更新时间都显示距现在过去多久时间。
<img width="143" alt="image" src="https://user-images.githubusercontent.com/53633824/161415091-da36410e-b166-45c2-bc91-09d7ad28a52f.png">
有没有一种办法能够显示出具体的发布date，比如 `2022-04-03`。

我自己做过很多尝试，都没有成功。在无cache刷新过程中，发现date format先是YYYY-MM-DD格式，之后变成了... ago的格式。猜测是javascript的效果。想问下哪里可以改这个设置。谢谢

**Comment by ppoffice** [comment, marked as answer]
修改这一行：https://github.com/ppoffice/hexo-theme-icarus/blob/master/source/js/main.js#L25

**Reply by realzza** [reply]
谢谢，已解决
```jsx
$(this).text(moment($(this).attr('datetime')).fromNow());
```
修改成
```jsx
$(this).text(moment($(this).attr('datetime')).format("YYYY-MM-DD"));
```
